tests/network/primary_pod_network.go

```diff
@@ -88,13 +88,13 @@ var _ = SIGDescribe("[Serial]Primary Pod Network", func() {
 				var vmi *v1.VirtualMachineInstance
 
 				BeforeEach(func() {
-					tmpVmi, err := newFedoraWithGuestAgent()
+					var err error
+					vmi, err = newFedoraWithGuestAgent()
```

Comment:
the `tmpVmi` was intentional, please leave it as is.
It assures that if the creation fails, we do not attempt to remove the vmi.

Reply:
so , the beginning of BeforeEach missing vmi = nil
well we have just one test here, and 2nd asserts are masked, but still